elements/pfe-autocomplete/demo/pfe-autocomplete.html

```diff
@@ -2,7 +2,7 @@
 <link rel="stylesheet" href="/core/pfe-styles/pfe-base.css">
 <link rel="stylesheet" href="/core/pfe-styles/pfe-context.css">
 <link rel="stylesheet" href="/core/pfe-styles/pfe-layouts.css">
-<link rel="stylesheet" href="/core/pfe-core/pfe.css">
+<link rel="stylesheet" href="/core/pfe-styles/pfe.css">
```

Comment:
ordinarily we'd want one commit per feature, but that won't work on this branch, unfortunately

this is annoying as heck, but in order to avoid conflicts while rebasing, these all need to be committed with a conventional commit scope like

```
docs(autocomplete): use pfe-styles
```

this is the shell script (fish shell, needs some adjustment for bash, probably) that I use for that

```fish
for p in (ls elements)
    set -lx SCOPE (string replace pfe- '' $p)
    git add "elements/pfe-$SCOPE"
    git cim "fix($SCOPE): use pfe-styles";
end
```

Reply:
I *think* the bash version looks like this:

```sh
for p in $(ls elements)
do
    set -x SCOPE=`${$p/pfe-/''}`
    git add "elements/pfe-$SCOPE"
    git cim "fix($SCOPE): use pfe-styles";
end
```